src/editor-settings.php

```diff
@@ -160,7 +160,19 @@ public function register_settings() {
 				'stackable_enable_carousel_lazy_loading',
 				array(
 					'type' => 'boolean',
-					'description' => __( 'Disables image lazy loading when using images inside carousel-type blocks to prevent space or layout issues .', STACKABLE_I18N ),
+					'description' => __( 'Disables image lazy loading when using images inside carousel-type blocks to prevent space or layout issues.', STACKABLE_I18N ),
+					'sanitize_callback' => 'sanitize_text_field',
+					'show_in_rest' => true,
+					'default' => true,
+				)
+			);
+
+			register_setting(
+				'stackable_editor_settings',
+				'stackable_inherit_styles_from_theme',
```

Comment:
For now, since this only affects buttons, please use a name that's specific for buttons. Please adjust where necessary.
```suggestion
				'stackable_inherit_button_styles_from_theme',
```